.github/workflows/cmake.yml

```diff
@@ -47,3 +51,14 @@ jobs:
     - name: Test
       working-directory: ${{github.workspace}}/build/tests
       run: ctest -C ${{env.BUILD_TYPE}}
+
+    - name: Build
+      if: matrix.config.os == 'windows-latest'
+      run: pyinstaller scripts\zeno_win.spec
```

Comment:
This will break linux CI, please move `pyinstaller` workflow to a separate `yml`, with only `windows-latest` in its matrix.